andela/activo-web
# #157469989 Display details of a logged-in user

#### What does this PR do?
Displays the details of a logged-in user

#### Description of task to be completed?
- create a user-detail component
- create an interface for the user-detail component
- abstract the props for the navbar into an interface
- create the auth reducer
- create the truncate name helper-function
- write test for the user-detail component
- modify the test for navbar component

#### How should this be manually tested?
- After cloning the repo, run `yarn install` to install the required dependencies.
- Run `yarn start:dev` to start the development server
- Log-in with your Andela email.
- Your Name and image should appear on the right-hand side of the Navigation bar.

#### Any background context you want to provide?
N/A

####What are the relevant pivotal tracker stories?

- [#157469989](https://www.pivotaltracker.com/story/show/157469989)

#### Screenshots
<img width="1280" alt="screen shot 2018-06-05 at 12 13 22 pm" src="https://user-images.githubusercontent.com/24798364/40994977-b10959bc-68f5-11e8-9256-63253ebe7d09.png">
